examples/api/lib/material/dropdown_menu/dropdown_menu.0.dart

```diff
@@ -106,28 +141,6 @@ class _DropdownMenuExampleState extends State<DropdownMenuExample> {
   }
 }
 
-enum ColorLabel {
-  blue('Blue', Colors.blue),
-  pink('Pink', Colors.pink),
-  green('Green', Colors.green),
-  yellow('Yellow', Colors.yellow),
-  grey('Grey', Colors.grey);
-
-  const ColorLabel(this.label, this.color);
-  final String label;
-  final Color color;
-}
-
-enum IconLabel {
-  smile('Smile', Icons.sentiment_satisfied_outlined),
-  cloud(
-    'Cloud',
-    Icons.cloud_outlined,
-  ),
-  brush('Brush', Icons.brush_outlined),
-  heart('Heart', Icons.favorite);
-
-  const IconLabel(this.label, this.icon);
-  final String label;
-  final IconData icon;
+void main() {
```

Comment:
I typically put the main at the top because in VSCode, when you go to run the test, you have to find the main to click the "Run" button. That's easier if the main is at the top.

Reply:
OK, main is at the top.